plugins/eth-validator-cli/plugin_eth-validator-cli.sh

```
@@ -0,0 +1,151 @@
+PLUGIN_SOURCE_PATH="$SOURCE_DIR"
+
+# Gets latest tag
+function getLatestVersion(){
+  TAG=$(curl -s $RELEASE_URL | jq -r .tag_name )
+}
+
+# Downloads latest release
+function downloadClient(){
+  local _custom
+  # Handle custom naming convention
+  [[ $_arch == "amd64" ]] && _custom="x64"
```

Shellcheck (0.10.0)

[warning] _arch is referenced but not assigned.

(SC2154)